docs/source/usage/workflows/numberOfCells.rst

```diff
@@ -8,10 +8,14 @@ Setting the Number of Cells
 Together with the grid resolution in :ref:`grid.param <usage-params-core>`, the number of cells in our :ref:`.cfg files <usage-tbg>` determine the overall size of a simulation (box).
 The following rules need to be applied when setting the number of cells:
 
-Each GPU needs to:
+Each device needs to:
 
 #. contain an integer *multiple* of supercells
 #. at least *three* supercells
+#. for non periodic boundaries the absorber must fit into the local volume
```

Comment:
I think the term `the absorber` is not something a user knows (but our slang).
maybe better write
```suggestion
#. for non periodic boundary conditions, the number of absorbing boundary cells for devices at the simulation boundary (see :ref:`grid.param <usage-params-core>`) must fit into the local volume
```